deployment/ccip/changeset/testhelpers/test_ton_helpers.go

```diff
@@ -0,0 +1,321 @@
+package testhelpers
+
+import (
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+	tonaddress "github.com/xssnick/tonutils-go/address"
+
+	"github.com/smartcontractkit/chainlink-ccip/pkg/types/ccipocr3"
+	cldf_ton "github.com/smartcontractkit/chainlink-deployments-framework/chain/ton"
+	"github.com/smartcontractkit/chainlink-deployments-framework/deployment"
+	cldf "github.com/smartcontractkit/chainlink-deployments-framework/deployment"
+
+	"github.com/smartcontractkit/chainlink-ccip/chains/evm/gobindings/generated/v1_6_0/onramp"
+	tonstate "github.com/smartcontractkit/chainlink/deployment/ccip/shared/stateview/ton"
+	commoncs "github.com/smartcontractkit/chainlink/deployment/common/changeset"
+)
+
+type TonTestDeployPrerequisitesChangeSet struct {
+	T                 *testing.T
+	TonChainSelectors []uint64
+}
+
+var _ commoncs.ConfiguredChangeSet = TonTestDeployPrerequisitesChangeSet{}
+
+func (c TonTestDeployPrerequisitesChangeSet) Apply(e cldf.Environment) (cldf.ChangesetOutput, error) {
+	t := c.T
+
+	tonChains, err := tonstate.LoadOnchainStateTon(e)
+	require.NoError(t, err)
```

Comment:
Do we have a good explanation why we inject and propagate the test context everywhere? 

What would happen if we just returned an error, as one would normally, and the client (e.g., a test) asserts no errors?